features/admin.email-management.v1/components/email-customization-footer.tsx

```diff
@@ -63,6 +70,9 @@ const EmailCustomizationFooter: FunctionComponent<EmailCustomizationFooterProps>
             padded={ true }
             data-componentid={ componentId }
         >
+            { tierName === TenantTier.FREE && (
```

Comment:
Shall we check whether this is returned as TenantTier.FREE in onprem as well? If yes, we may need to introduce a config to prevent this banner from being shown in onprem console